modules/runtime-info/src/main/java/org/opencastproject/runtimeinfo/rest/RestDocData.java

```diff
@@ -21,6 +21,8 @@
 
 package org.opencastproject.runtimeinfo.rest;
 
+import static org.apache.commons.lang3.StringUtils.isBlank;
```

Comment:
Not to be picky, but this isn't needed right?  I'm surprised that checkstyles isn't complaining here.

Reply:
It is as `DocData` no longer has a `isBlank` implementation. This class uses it.